codalab/scripts/initialize_from_fixture.py

```diff
@@ -0,0 +1,14 @@
+import os, json
+
+with open("apps/web/fixtures/initialize_site.json",'r') as initial_site_json:
+    json_data = json.loads(initial_site_json.read())
+    json_data[0]['fields']['domain'] = os.environ['CODALAB_SITE_DOMAIN']
+    json_data[0]['fields']['name'] = 'CODALAB_SITE_DOMAIN'
```

Comment:
Probably change `os.environ` -> `os.environ.get` with a default like localhost in-case it's not supplied.

Also it looks like line 6 needs to be changed to either the same environment variable, or a different environment variable for the site "name".

![image](https://user-images.githubusercontent.com/28552312/156096250-0e67c0b6-de45-4d5f-9938-ae4124319364.png)